CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 ## Unreleased
 
+### Features
+
+- Adds metrics ([#5402](https://github.com/getsentry/sentry-react-native/pull/5402))
```

Comment:
We could mention that the feature is beta (similar to [feedback](https://github.com/getsentry/sentry-react-native/blob/ff5a06a17adf188a929feb79fd9f91219e602ce4/CHANGELOG.md?plain=1#L1156) and [SR](https://github.com/getsentry/sentry-react-native/blob/ff5a06a17adf188a929feb79fd9f91219e602ce4/CHANGELOG.md?plain=1#L2153) in the past)
```suggestion
- Adds Metrics Beta ([#5402](https://github.com/getsentry/sentry-react-native/pull/5402))
```

Reply:
also a small snippet on how to use it would be nice here

Reply:
I would advocate towards shipping as beta since the product is still labeled [as beta in JS sidebar](https://docs.sentry.io/platforms/javascript/metrics/)